projects/assets/src/translations/en/product.ts

```diff
@@ -21,6 +21,7 @@ export const product = {
     showLess: 'Show less...',
     showMore: 'Show more...',
     sortBy: 'Sort by',
+    sortResults: 'Sort results',
```

Comment:
Where is this used?  I must have missed it.

Reply:
Yes it is used in sorting.comp.html:
<img width="533" alt="Screen Shot 2022-03-14 at 3 17 23 PM" src="https://user-images.githubusercontent.com/49131189/158244901-557cc810-1cff-4369-982a-87d7490caeee.png">

Reply:
Ah! There it is!